### ISSUE TITLE
Screen reader does not announce up/down to navigate through a shell's history

### ISSUE THREAD

**Issue by Tyriar** [xtermjs/xterm.js#1960]
Repro:

- `ls`, ls is announced
- enter, the output is announced
- up, "ls" is _not_ announced

Code pointer:

https://github.com/xtermjs/xterm.js/blob/f41146fb40e5bc089998c9772e387dde4a83589e/src/AccessibilityManager.ts#L79
